content/en/docs/demo/services/accounting.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Accounting Service
 linkTitle: Accounting
-aliases: [accountingservice]
+aliases: [accounting]
```

Comment:
Aliases are "forever". This must not be renamed:

```suggestion
aliases: [accountingservice]
```

Please revert the changes to `aliases` fields in this and other pages.